docs/sphinx/source/pvsystem.rst

```diff
@@ -90,71 +107,200 @@ Multiple methods may pull data from the same attribute. For example, the
+:py:meth:`~pvlib.pvsystem.Array.get_aoi`. The angle of incidence (AOI)
+calculations require `surface_tilt`, `surface_azimuth` and the extrinsic
+sun position. The `PVSystem` method :py:meth:`~pvlib.pvsystem.PVSystem.get_aoi`
+uses the `surface_tilt` and `surface_azimuth` attributes from the
+:py:class:`pvlib.pvsystem.PVSystem` instance, and so requires only `solar_zenith`
+and `solar_azimuth` as arguments.
+
+.. ipython:: python
+
+    # single south-facing array at 20 deg tilt
+    system_one_array = pvsystem.PVSystem(surface_tilt=20, surface_azimuth=180)
+    print(system_one_array.surface_tilt, system_one_array.surface_azimuth)
+
+    # call get_aoi with solar_zenith, solar_azimuth
+    aoi = system_one_array.get_aoi(solar_zenith=30, solar_azimuth=180)
+    print(aoi)
 
-The first PVSystem parameters are `surface_tilt` and `surface_azimuth`.
-These parameters are used in PVSystem methods such as
-:py:meth:`~pvlib.pvsystem.PVSystem.get_aoi` and
-:py:meth:`~pvlib.pvsystem.PVSystem.get_irradiance`. Angle of incidence
-(AOI) calculations require `surface_tilt`, `surface_azimuth` and also
-the sun position. The :py:meth:`~pvlib.pvsystem.PVSystem.get_aoi` method
-uses the `surface_tilt` and `surface_azimuth` attributes in its PVSystem
-object, and so requires only `solar_zenith` and `solar_azimuth` as
-arguments.
+
+The `Array` method :py:meth:`~pvlib.pvsystem.Array.get_aoi`
+operates in a similar manner.
 
 .. ipython:: python
 
-    # 20 deg tilt, south-facing
-    system = pvsystem.PVSystem(surface_tilt=20, surface_azimuth=180)
-    print(system.surface_tilt, system.surface_azimuth)
+    # two arrays each at 30 deg tilt with different facing
+    array_one = pvsystem.Array(surface_tilt=30, surface_azimuth=90)
+    array_one_aoi = array_one.get_aoi(solar_zenith=30, solar_azimuth=180)
+    print(aoi)
 
+
+The `PVSystem` method :py:meth:`~pvlib.pvsystem.PVSystem.get_aoi`
+operates on all `Array` instances in the `PVSystem`, whereas the the
+`Array` method operates only on its `Array` instance.
+
+.. ipython:: python
+
+    array_two = pvsystem.Array(surface_tilt=30, surface_azimuth=220)
+    system_multiarray = pvsystem.PVSystem(arrays=[array_one, array_two])
+    print(system_multiarray.num_arrays)
     # call get_aoi with solar_zenith, solar_azimuth
-    aoi = system.get_aoi(30, 180)
+    aoi = system_multiarray.get_aoi(solar_zenith=30, solar_azimuth=180)
+    print(aoi)
+
+
+As a reminder, when the PV system includes more than one array, the output of the
+`PVSystem` method :py:meth:`~pvlib.pvsystem.PVSystem.get_aoi` is a *tuple* with
+the order of the elements corresponding to the order of the arrays.
+
+Other `PVSystem` and `Array` methods operate in a similar manner. When a `PVSystem`
+method needs input for each array, the input is provided in a tuple:
+
+.. ipython:: python
+
+    aoi = system.get_aoi(solar_zenith=30, solar_azimuth=180)
     print(aoi)
+    system_multiarray.get_iam(aoi)
```

Comment:
Not sure how to achieve that without muddying the example. `print(np.array(1))` will unwrap the value, but if you put arrays in a tuple you get the same result as above. We would need to resort to mapping `str` over the tuple, then printing it.

I guess `tuple.__str__` calls `__repr__` on its entries, not `__str__`. Seems like odd behavior to me, but what can you do.

```suggestion
    iam = system_multiarray.get_iam(aoi)
    print(tuple(map(str, iam)))
```

Reply:
Ah I guess I was thinking that, if there was an issue to fix, it was that `system.get_iam` returned tuple of array instead of tuple of float, not that the printing here needs to be fixed.  Let's not worry about this.

Reply:
Might be worth a separate issue to address the return types from the function layer.